### DISCUSSION TITLE
Ringtone at group call

### DISCUSSION THREAD

**Discussion by armbck** [element-hq/element-meta#881] [Ideas]
#### What would you like to do?
To be able to call multiple participants so that their devices sound a ringtone.

#### Why would you like to do it?
To inform users there is a call starting. To be able to call group of people without having to inform them separately using another method of communication.

#### How would you like to achieve it?
Can you add the feature or inform how to enable it, if it is already implemented. We could not find the feature in demo either.

**Comment by msulmanzig** [comment]
I need help in this.
